packages/common/src/static/referencesTypes.ts

```diff
@@ -0,0 +1,32 @@
+export const REFERENCES_TYPES_IDS = {
+  BRGM: 'brg',
+  DEAL: 'dea',
+  DEB: 'deb',
+  DGEC: 'dge',
+  Ifremer: 'ifr',
+  IRSN: 'irs',
+  "Nom d'usage": 'nus',
```

Comment:
```suggestion
  NomDUsage: 'nus',
```

Comme ça on n'a pas d'accès bizarre à REFERENCES_TYPES_IDS (J'ai commencé à changer ça pour les substancesFiscales, ça change la vie)